tests/unit/services/cookies-test.js

```diff
@@ -74,6 +74,11 @@ describe('CookiesService', function() {
         expect(afterRoundtrip).to.eq(value);
       });
 
+      it('handles invalid values for cookies', function() {
+        document.cookie = '=blank';
+        expect(this.subject().read('')).to.be.blank;
```

Comment:
Sorry - I have one more nitpick: this should actually be `undefined`, not `blank`:

```js
expect(this.subject().read('')).to.be.undefined;
```